From dfd077a3620b58b665d2683e0a9fe13f006e99c1 Mon Sep 17 00:00:00 2001
From: Lukas Woodtli <lukas.woodtli@husqvarnagroup.com>
Date: Thu, 10 Apr 2025 15:31:42 +0200
Subject: [PATCH] Use optimization level 2 as default for Python

This is as if the command line option `-OO` is set implicitly.

Upstream-Status: Inappropriate [embedded-specific]
---
 Python/initconfig.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Python/initconfig.c b/Python/initconfig.c
index 192089b..3f91fac 100644
--- a/Python/initconfig.c
+++ b/Python/initconfig.c
@@ -788,7 +788,7 @@ config_init_defaults(PyConfig *config)
     config->bytes_warning = 0;
     config->inspect = 0;
     config->interactive = 0;
-    config->optimization_level = 0;
+    config->optimization_level = 2;
     config->parser_debug= 0;
     config->write_bytecode = 1;
     config->verbose = 0;
